.gitattributes

```diff
@@ -1,5 +1,29 @@
+#
+# Exclude these files from release archives.
+# This will also make them unavailable when using Composer with `--prefer-dist`.
+# https://blog.madewithlove.be/post/gitattributes/
+#
 /tests export-ignore
-/bin export-ignore
-/.gitignore export-ignore
-/.travis.yml export-ignore
-/phpunit.xml.dist export-ignore
+/psalm export-ignore
+/.github export-ignore
+.gitattributes export-ignore
+.gitignore export-ignore
+.phpcsignore export-ignore
+.travis.yml export-ignore
+codecov.yml export-ignore
+phpunit.xml.dist export-ignore
+psalm.xml export-ignore
+verify.txt export-ignore
+
+#
+# Auto detect text files and perform LF normalization
+# http://davidlaing.com/2012/09/19/customise-your-gitattributes-to-become-a-git-ninja/
+#
+* text=auto
+
+#
+# The above will handle all files NOT found below
```

Comment:
Why did we need to add this section?

Reply:
Just a minor quality improvement if the package is pulled via Composer.